### PR TITLE
Package ocsigen-start.4.5.0

### PR DESCRIPTION
**Pull request by jrochel**
### `ocsigen-start.4.5.0`
An Eliom application skeleton ready to use to build your own application with users, (pre)registration, notifications, etc
Ocsigen Start is a set of higher-level libraries for building client-server web applications with Ocsigen (Js_of_ocaml and Eliom).
It provides modules for user management (session management, registration, activation keys, ...), managing groups of users, displaying tips, and easily sending notifications to the users.
Ocsigen Start comes with an eliom-distillery template for an app with a database, user management, and session management.
This template is intended to serve as a basis for quickly building the Minimum Viable Product for web applications with users.
The goal is to enable the programmer to concentrate on the core of the app, and not on user management.

---
* Homepage: https://ocsigen.org/ocsigen-start/
* Source repo: git+https://github.com/ocsigen/ocsigen-start.git
* Bug tracker: https://github.com/ocsigen/ocsigen-start/issues

---
:camel: Pull-request generated by opam-publish v2.1.0

### CODE REVIEW

**Comment by jrochel** [comment]
@kit-ty-kate I think that this PR can be merged. The compilation errors of js_of_ocaml are due to https://github.com/ocsigen/js_of_ocaml/issues/1182#event-5683475208.

File: packages/ocsigen-start/ocsigen-start.4.1.0/opam [hunk 1]
```diff
@@ -15,7 +15,7 @@ depends: [
   "pgocaml_ppx" {>= "4.0"}
   "safepass" {>= "3.0"}
   "ocsigen-i18n" {>= "3.7.0" & < "4.0.0"}
-  "eliom" {>= "7.0.0"}
+  "eliom" {>= "7.0.0" & < "9.0.0"}
```
Comment (kit-ty-kate) [inline packages/ocsigen-start/ocsigen-start.4.1.0/opam:18]: I’m not seeing any issues with eliom 9.0.0 here. Did you mean 10.0.0?

Comment (jrochel) [reply packages/ocsigen-start/ocsigen-start.4.1.0/opam:18]: No, 9.0.0 is what I meant. It is not a compilation issue, but a run-time issue: https://github.com/ocsigen/ocsigen-start/issues/638
Thanks for checking in though.

Comment (kit-ty-kate) [reply packages/ocsigen-start/ocsigen-start.4.1.0/opam:18]: I see. Faire enough

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**